apps/script/HoC2023ScriptFiles/README.md

```diff
@@ -0,0 +1,98 @@
+# HoC 2023 Script Files
+
+Run the `HoC2023AiGenerateWeights.py` to generate the associated output weights contained in files like cached_background_effects_map.json that are used to calculate final effects output for the HoC 2023 activity.
+
+Before running the script, you must assign the openai.api_key with your Open AI API key
+the in Hoc2023AiHelperFunctions.py file.
+Run `HoC2023AiGenerateWeights.py` from the code-dot-org root directory using the following command:
+`python apps/script/HoC2023ScriptFiles/HoC2023AiGenerateWeights.py`
+
+Note that the script generates/updates 3 files in apps/static/dance/ai/model:
+- cached_background_effects_map.json
+- cached_foreground_effects_map.json
+- cached_palettes_map.json
+
+These files store association values measuring the similarity between an emoji and each output.
+Embeddings from Open AI's Ada model are used to compute these values.
+An embedding is a vector (list) of floating point numbers, and embeddings are used to
+measure the relatedness of text strings. The distance between two vectors measures
+their relatedness. https://platform.openai.com/docs/guides/embeddings
+We also store embeddings in caches files as as pickle files, python's native way to serialize data.
+
+Below are background effects, palettes and foreground effects in the order that they appear in the dropdowns. When this file is run and the three cached data files are replaced in apps/static/dance/ai/model, please update the lists below with the output from this script.
+
+BACKGROUND EFFECTS
```

Comment:
These are for some reason all showing up in a single line when viewed on github (maybe being interpreted as a markdown table or something)?

Reply:
Thanks! Need to add two spaces to create newline. Will do so via print function - thanks!